clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1010,48 +1010,109 @@ architecture to achieve optimal efficiency.  Google is sharing
+  lists can be obtained by walking the #instr_t operands with instr_get_dst() and
+  instr_get_src().
+- ISA mode: is always #DR_ISA_REGDEPS.  This information can be obtained using
+  instr_get_isa_mode().
+- encoding bytes: an array of bytes containing the #DR_ISA_REGDEPS #instr_t
+  encoding.  Note that this information is present only for decoded instructions
+  (i.e., #instr_t generated by decode() or decode_from_copy()).  This information
+  can be obtained using instr_get_raw_bits().
+- length: the length of the encoded instruction in bytes.  Note that this
+  information is present only for decoded instructions (i.e., #instr_t generated by
+  decode() or decode_from_copy()).  This information can be obtained by accessing
+  the #instr_t length field.
+
+Note that all routines that operate on #instr_t and #opnd_t are also supported for
+#DR_ISA_REGDEPS instructions.  However, querying information outside of those
+described above (e.g., the instruction opcode with instr_get_opcode()) will return
+the zeroed value set by instr_create() or instr_init() when the #instr_t was
+created (e.g., instr_get_opcode() would return OP_UNDECODED).
+
+On top of instructions and memory acceses, traces also have
+#dynamorio::drmemtrace::trace_marker_type_t markers.
+All markers of the original trace are present, except for:
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_SYSCALL_IDX
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_SYSCALL
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_SYSCALL_TRACE_START
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_SYSCALL_TRACE_END
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_SYSCALL_FAILED, which have been removed.
+Because tracing overhead results into inflated context switches, the
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_CPU_ID values have been modified to
+"unknown CPU" to avoid confusion. We recommend users to use our scheduler for a
+realistic schedule of a trace threads. Also, the only
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_ID
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_ARG
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_RETVAL
+#dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_RETADDR markers preserved are those
+related to SYS_futex functions.
+
+Finally, every trace has a v2p.textproto file associated to it, which provides a
+plausible virtual to physical mapping of the virtual addresses present in a trace
+for more realistic TLB simulations. This is a static virtual to physical mapping
+with 2 MB pages. Users can generate different mappings (e.g., smaller page size)
+by modifying such file, or create their own mapping following the same
+v2p.textproto format.
 
 \section sec_google_get Getting the Traces
 
 The Google Workload Traces can be downloaded from:
 
- - [Google workload trace folder](https://console.cloud.google.com/storage/browser/external-traces)
+ - [Google workload trace folder](TODO: add new link to Google Storage Bucket once known)
```

Comment:
I assume we'd use the same bucket?  Should we make version_1 and version_2 subdirs?  Or just delete v1?  Maybe just delete.

Reply:
Let's discuss offline.

Reply:
The new one could be sthg like `console.cloud.google.com/storage/browser/external-traces-v2`.

Reply:
That's right, I created: https://console.cloud.google.com/storage/browser/external-traces-v2 (now empty).
I've added it as link to the new public traces.
I also added a section about deprecated public v1 traces with the old link to them.

I think it might be a good idea to also have DOI for citations? We could make one with https://zenodo.org/ . This way it's easy to track who/how many cite this work.